systems/subscriber/registry/cmd/server/main.go

```diff
@@ -75,8 +74,6 @@ func runGrpcServer(gormdb sql.Db) {
 		instanceId = inst.String()
 	}
 
-	networkClient := creg.NewNetworkClient(serviceConfig.RegistryHost)
```

Comment:
We need a network client in order to validate the provided network against all the remote networks for the org.